programmers/난이도별/level02.프린터/dkdlelk99.py

```diff
@@ -0,0 +1,16 @@
+def solution(priorities, location):
+    temp = []
+    for i in range(len(priorities)):
+        temp += [[i,priorities[i]]]
+
+    while temp[0][1] != max(priorities):
+        for i in range(1,len(priorities)):
+            if temp[0][1] < temp[i][1]:
+                temp.append(temp[0][1])
+                temp.remove(temp[0][1])
```

Comment:
```suggestion
```suggestion 
    prioritiesMax = max(priorities)
    prioritiesLen = len(priorities)
    while temp[0][1] != prioritiesMax:
        for i in range(1,prioritiesLen):
            if temp[0][1] < temp[i][1]:
                temp.append(temp[0][1])
                temp.remove(temp[0][1])
```
배열을 다루는 함수 (ex: len, max) 는 탐색에 대한 시간이 소요됩니다!
즉, 반복문을 돌 때 마다 len과 max 연산을 해야 되는 것입니다..

그래서 결과값을 변수로 선어해놓는 게 좋습니다.